src/pages/user/order/OrderBasket.tsx

```diff
@@ -48,43 +67,51 @@ function OrderBasket() {
   const workspaceId = searchParams.get('workspaceId');
   const tableNo = searchParams.get('tableNo');
 
-  const basketRef = React.useRef<HTMLDivElement>(null);
-
   if (orderBasket.length == 0) {
     navigate(-1);
   }
 
-  useEffect(() => {
-    basketRef.current?.scroll({ top: 0, behavior: 'smooth' });
-  }, []);
+  const flushOrderBasket = () => {
```

Comment:
```suggestion
  const clearOrderBasket = () => {
```

그리고 전체삭제도 confirm창 띄워야할 것 같아요